modules/test/playwright/tests/analytics-web/supportLiferayCookieManager.spec.ts

```diff
@@ -0,0 +1,203 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginAnalyticsCloudTest} from '../../fixtures/loginAnalyticsCloudTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {liferayConfig} from '../../liferay.config';
+import {
+	connectToAnalyticsCloud,
+	disconnectFromAnalyticsCloud,
+	goToAnalyticsCloudInstanceSettings,
+	syncAllContacts,
+	syncSite,
+} from '../analytics-settings-web/utils/analyticsSettings';
+import {createDataSource} from '../osb-faro-web/utils/dataSource';
+
+export const test = mergeTests(
+	loginAnalyticsCloudTest(),
+	loginTest(),
+	featureFlagsTest({
+		'LPD-10588': true,
+	})
+);
+
+async function changeCookiePreference(
+	page,
+	{enableCookieBanner, enableExplicitCookieConsentMode}
+) {
+	await page.getByLabel('Open Applications MenuCtrl+Alt+A').click();
+
+	await page.getByRole('tab', {name: 'Control Panel'}).click();
+
+	await page.getByRole('menuitem', {name: 'Instance Settings'}).click();
+
+	await page.getByRole('link', {name: 'Cookies'}).click();
+
+	if (enableCookieBanner) {
+		await page.getByLabel('Enabled').check();
+	} else {
+		await page.getByLabel('Enabled').uncheck();
+	}
+
+	if (enableExplicitCookieConsentMode) {
+		await page.getByLabel('Explicit Cookie Consent Mode').check();
+	} else {
+		await page.getByLabel('Explicit Cookie Consent Mode').uncheck();
+	}
+
+	const submitButton = await page.$(
+		'button[data-qa-id="submitConfiguration"]'
+	);
+
+	await submitButton.click();
+
+	await page.waitForTimeout(3000);
+}
+
+async function connectACToDXP(page) {
+	await createDataSource(page);
+
+	await goToAnalyticsCloudInstanceSettings(page);
+
+	const cookieBannerElement = await page.$('div.portlet-cookies-banner');
+
+	if (cookieBannerElement) {
+		await cookieBannerElement.evaluate((div) => {
+			div.style.display = 'none';
+		});
+	}
+
+	await disconnectFromAnalyticsCloud(page);
+
+	await connectToAnalyticsCloud(page);
+
+	await syncSite(page);
+
+	await syncAllContacts(page);
+
+	await page.getByRole('button', {name: 'Finish'}).click();
+}
```

Comment:
I have a different step that it is only works on my test. And also I'm already importing steps from analyticsSettings.

Reply:
Let's keep it as it is.

Reply:
Does it only work in your test? Why?

Reply:
Because I'm using this piece of code that doesn't make sense in other scenarios

```
	const cookieBannerElement = await page.$('div.portlet-cookies-banner');

	if (cookieBannerElement) {
		await cookieBannerElement.evaluate((div) => {
			div.style.display = 'none';
		});
	}
```

Reply:
Ohhh yeah, because only in this file do you enable the banner cookie